presqt/osf/classes/storage_folder.py

```diff
@@ -2,12 +2,12 @@
 from presqt.osf.classes.file import File
 
 class ContainerMixin:
-    def get_assets_objects(self, container):
+    def get_resources_objects(self, container):
```

Comment:
Seems like maybe there should be `storage.py` and `folder.py` like there is `project.py`